README.md

```diff
@@ -58,157 +58,81 @@ Who want to submit the output needs to send the pull-request to this repository.
 
-Modify `conformance-suite/docker-compose.xml` as follows.
+### Test Reports
 
-**Note: We need to set `fintechlabs.base_url` with public FQDN to change from `https://localhost:8443`** 
+Once `test_runner` service has finished and exited, test reports will be copied to the 
+[./conformance-suite/report](./report) directory.
 
-```
-       context: ./server-dev
-     volumes:
-      - ./target/:/server/
--    command: java -jar /server/fapi-test-suite.jar --fintechlabs.devmode=true --fintechlabs.startredir=true
-+    command: java -jar /server/fapi-test-suite.jar --fintechlabs.devmode=true --fintechlabs.startredir=true --fintechlabs.base_url=https://${CONFORMANCE_SUITE_FQDN}
-     links:
-      - mongodb:mongodb
-      - microauth:microauth
-```
 
-Build FAPI Conformance suite server and boot the all containers using Docker Compose.
-
-```
-cd conformance-suite
-mvn clean package
-docker-compose up -d
-```
+### Modify your `hosts` file
 
-Generate server certificates, Keycloak realm config and FAPI Conformance suite configs with your FQDN.
+To access to Keycloak and Resource server with FQDN, modify your `hosts` file in your local machine as follows.
 
 ```
-cd ../keycloak-fapi
-./setup-fqdn.sh
+127.0.0.1 as.keycloak-fapi.org rs.keycloak-fapi.org conformance-suite.keycloak-fapi.org
 ```
 
-Boot the containers using Docker Compose.
+### Run FAPI Conformance test plan manually
 
-```
-docker-compose up -d
-```
+1. Run this project with `AUTOMATE_TESTS=false` environment variable set
+2. Open https://conformance-suite.keycloak-fapi.org
+3. Click `Create a new test plan` button.
+4. Choose `FAPI-RW-ID2 (and OpenBankingUK): Authorization server test (latest version)` as Test Plan.
+5. Choose `Client Authentication Type` you want to test.
+6. Choose `plain_fapi` as FAPI Profile.
+7. Choose `plain_response` as FAPI Response Mode.
+8. Click `JSON` tab and paste content of the configuration.
+  * If you want to use private_key_jwt client authentication, use [fapi-conformance-suite-configs/fapi-rw-id2-with-private-key-PS256-PS256.json](./fapi-conformance-suite-configs/fapi-rw-id2-with-private-key-PS256-PS256.json) or [fapi-conformance-suite-configs/fapi-rw-id2-with-private-key-ES256-ES256.json](./fapi-conformance-suite-configs/fapi-rw-id2-with-private-key-ES256-ES256.json).
+  * If you want to use mtls client authentication, use [fapi-conformance-suite-configs/fapi-rw-id2-with-mtls-PS256-PS256.json](./fapi-conformance-suite-configs/fapi-rw-id2-with-mtls-PS256-PS256.json) or [fapi-conformance-suite-configs/fapi-rw-id2-with-mtls-ES256-ES256.json](./fapi-conformance-suite-configs/fapi-rw-id2-with-mtls-ES256-ES256.json).
+9. Click `Create Test Plan` button and follow the instructions. To proceed with the tests, You can authenticate using `john` account with password `john`. When rejecting authentication scenario, you can use `mike` account with password `mike`. In this case, you need to click `No` button to cancel the authentication in the consent screen.
```

Comment:
I've tried the manual tests by setting `AUTOMATE_TESTS=false`. According to the README, I've expected that I myself operate the browser manually for login and authorizing the client. However, operating the browser has still be done automatically. IMO, this operation might be done manually, not automatically when specifying `AUTOMATE_TESTS=false`, which can provide also full manual conformance test as usual.

Reply:
@tnorimat I have added a separate json config file for automation. This brought my attention to the OpenID conformance suite remembering the last config used. I didn't see the need for a persistent mongodb volume, especially for pipeline use, however I left a note in the README for those that may want the persistent volume for manual use. WDYT?

Now, I believe this will work as you expect for manually running the test suite.

Reply:
@brothwellIntegral Thank you for clarifying the point. I understood.